python-web-scraping-practical-introduction/mechanicalsoup.py

```diff
@@ -0,0 +1,14 @@
+import time
+import mechanicalsoup
```

Comment:
Applying `isort` adds a space between the stdlib module and the external package:

```suggestion

import mechanicalsoup
```